### ISSUE TITLE
[UX IMPROVEMENT] Make the header and Left navigation sticky 

### ISSUE THREAD

**Issue by nloureiro** [ethereum/geth-website#181]
To improve the navigation experience on the different pages, the header and left navigation should be sticky to the browser window.

This is a design good practice that is used in almost all Devdocs websites.

Header: Add background color and make it fixed on desktop and mobile

Left Navigation: Add background color,  make it fixed on desktop and mobile, and add overflow to scroll when the content doesn't fit the screen vertically.

Use-case example, using devtools in the browser to produce these screencasts

https://user-images.githubusercontent.com/1120748/208143113-5398b77d-0c09-442d-9db7-32b23f19f84a.mov

https://user-images.githubusercontent.com/1120748/208143251-f1f8f4c9-bd12-4b7c-b28c-e598918ce0b9.mov

cautions to the top border on the left navigation
![Screen Shot 2022-12-16 04 19 39 PM](https://user-images.githubusercontent.com/1120748/208142638-7c42a9f8-0a80-4591-a2aa-0a8054729d9a.png)

The content going under the header is on purpose.
![Screen Shot 2022-12-16 04 25 13 PM](https://user-images.githubusercontent.com/1120748/208144245-8538c6fa-53f7-483b-808f-94b746474d42.png)

**Comment by samajammin** [comment]
> The content going under the header is on purpose.

This looks a bit awkward to me. Should we consider making the color above the site nav solid (so the content doesn't show through ABOVE the site nav)?

**Comment by corwintines** [comment]
Agree, should just make the background there the background color so it looks less awkrward

**Comment by nloureiro** [comment]
> > The content going under the header is on purpose.
> 
> This looks a bit awkward to me. Should we consider making the color above the site nav solid (so the content doesn't show through ABOVE the site nav)?

actually, I like it. I would add that a very subtle transparency would make it even better.

Did a small screencast to illustrate it.

https://user-images.githubusercontent.com/1120748/208393648-1594294f-4992-4743-9693-e38ce2c64076.mov

**Comment by nhsz** [comment]
having text content between the header and navigation menu feels odd

<img width="356" alt="Screen Shot 2022-12-19 at 13 33 07" src="https://user-images.githubusercontent.com/948922/208474356-b15a89fa-ca14-4a12-ad56-c5c65e343d81.png">

having the content of the right menu below the header doesn't look good to me

<img width="786" alt="Screen Shot 2022-12-19 at 13 30 34" src="https://user-images.githubusercontent.com/948922/208473829-dabae2af-c890-4f10-b000-dfbde5fdc648.png">

I agree we should have a solid/blur background there.

**Comment by nloureiro** [comment]
> having text content between the header and navigation menu feels odd
> 
> <img alt="Screen Shot 2022-12-19 at 13 33 07" width="356" src="https://user-images.githubusercontent.com/948922/208474356-b15a89fa-ca14-4a12-ad56-c5c65e343d81.png">
> 
> having the content of the right menu below the header doesn't look good to me
> 
> <img alt="Screen Shot 2022-12-19 at 13 30 34" width="786" src="https://user-images.githubusercontent.com/948922/208473829-dabae2af-c890-4f10-b000-dfbde5fdc648.png">
> 
> I agree we should have a solid/blur background there.


That screenshot looks odd, yes. 

But with fullscreen mobile and with the user intention to scroll,  I like it. and if we add the 5% transparency, it's even better

check this Figma prototype
https://www.figma.com/proto/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?page-id=2848%3A21050&node-id=2857%3A21520&viewport=1157%2C757%2C1&scaling=min-zoom
